challenges/objects-arrays.js

```diff
@@ -55,33 +136,100 @@ Name email@example.com
-{"animal_name":"White-cheeked pintail","population":1,"scientific_name":"Anas bahamensis","state":"Oregon"},
-{"animal_name":"Black-backed jackal","population":2,"scientific_name":"Canis mesomelas","state":"Washington"},
-{"animal_name":"Brolga crane","population":9,"scientific_name":"Grus rubicundus","state":"New Mexico"},
-{"animal_name":"Common melba finch","population":5,"scientific_name":"Pytilia melba","state":"Pennsylvania"},
-{"animal_name":"Pampa gray fox","population":10,"scientific_name":"Pseudalopex gymnocercus","state":"Connecticut"},
-{"animal_name":"Hawk-eagle, crowned","population":10,"scientific_name":"Spizaetus coronatus","state":"Florida"},
-{"animal_name":"Australian pelican","population":5,"scientific_name":"Pelecanus conspicillatus","state":"West Virginia"}];
+zooAnimals = [{
+    "animal_name": "Jackal, asiatic",
+    "population": 5,
+    "scientific_name": "Canis aureus",
+    "state": "Kentucky"
+  },
+  {
+    "animal_name": "Screamer, southern",
+    "population": 1,
+    "scientific_name": "Chauna torquata",
+    "state": "Alabama"
+  },
+  {
+    "animal_name": "White spoonbill",
+    "population": 8,
+    "scientific_name": "Platalea leucordia",
+    "state": "Georgia"
+  },
+  {
+    "animal_name": "White-cheeked pintail",
+    "population": 1,
+    "scientific_name": "Anas bahamensis",
+    "state": "Oregon"
+  },
+  {
+    "animal_name": "Black-backed jackal",
+    "population": 2,
+    "scientific_name": "Canis mesomelas",
+    "state": "Washington"
+  },
+  {
+    "animal_name": "Brolga crane",
+    "population": 9,
+    "scientific_name": "Grus rubicundus",
+    "state": "New Mexico"
+  },
+  {
+    "animal_name": "Common melba finch",
+    "population": 5,
+    "scientific_name": "Pytilia melba",
+    "state": "Pennsylvania"
+  },
+  {
+    "animal_name": "Pampa gray fox",
+    "population": 10,
+    "scientific_name": "Pseudalopex gymnocercus",
+    "state": "Connecticut"
+  },
+  {
+    "animal_name": "Hawk-eagle, crowned",
+    "population": 10,
+    "scientific_name": "Spizaetus coronatus",
+    "state": "Florida"
+  },
+  {
+    "animal_name": "Australian pelican",
+    "population": 5,
+    "scientific_name": "Pelecanus conspicillatus",
+    "state": "West Virginia"
+  }
+];
 
 // The zoos need a list of all their animal's names converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
 const lowerCase = [];
-console.log(lowerCase); 
+const lower = (item, index) => {
+  return zooAnimals[index].animal_name.toLowerCase();
+}
+lowerCase.push(zooAnimals.map(lower))
+console.log(lowerCase);
 
 // The zoos need to know their total animal population across the United States.  Add up all the population numbers from all the zoos using the .reduce() method.
 const populationTotal = [];
+const add = (acc, item) => {
+  return acc + item.population;
+}
+populationTotal.push(zooAnimals.reduce(add, zooAnimals[0].population))
```

Comment:
Reduce does need an initial value for the accumulator but by using `zooAnimals[0].population` the first population is being added twice. Consider using 0.